app/models/user.rb

```diff
@@ -0,0 +1,33 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] kindly make sure the application asks for confirmation of email. kindly you need to add the `:confirmable`